modules/core/src/main/java/org/apache/ignite/internal/processors/metric/impl/MetricUtils.java

```diff
@@ -73,6 +73,19 @@ public static String metricName(String... names) {
         return String.join(SEPARATOR, names);
     }
 
+    /**
+     * Splits full metric name to registry name and metric name.
+     *
+     * @param name Full metric name.
+     * @return Array consist of registry name and metric name.
+     */
+    public static String[] fromFullName(String name) {
```

Comment:
Why not "IgniteBiTuple<String, String>"? Array return type is confusing.
I'm not necessarily asking for the change here, just curious.

Reply:
Refactored to `IgniteBiTuple`.